visualize-admin/visualization-tool
# feat: Add temporal entity dimension filtering to dashboard filters

...and fix the time range brush for `TemporalEntityDimension` in column charts.

Closes #1555

## How to test
### Time brush for temporal entity dimension based X axis
**PR**
1. Go to [this link](https://visualization-tool-git-fix-temporal-entity-filtering-ixt1.vercel.app/en/create/new?cube=https://agriculture.ld.admin.ch/foag/cube/MilkDairyProducts/Consumption_Price_Month&dataSource=Int).
2. Open Horizontal Axis settings.
3. Enable interactive filter.
4. ✅ See that it's possible to filter the chart.

**TEST**
1. Go to [this link](https://test.visualize.admin.ch/en/create/new?cube=https://agriculture.ld.admin.ch/foag/cube/MilkDairyProducts/Consumption_Price_Month&dataSource=Int).
2. Open Horizontal Axis settings.
3. Enable interactive filter.
4. ❌ See that it's not possible to filter the chart (NaN values displayed in the brush).

### Temporal entity dimension in shared dashboard filters
**PR**
1. Go to [this link](https://visualization-tool-git-fix-temporal-entity-filtering-ixt1.vercel.app/en/create/new?cube=https://agriculture.ld.admin.ch/foag/cube/MilkDairyProducts/Consumption_Price_Month&dataSource=Int).
2. Add another chart (e.g. line).
3. Proceed to layout options.
4. Switch to a dashboard layout.
5. Enable shared Date filter.
6. Filter the chart using the time brush.
7. ✅ See that both charts are filtered as expected.

**TEST**
1. Go to [this link](https://test.visualize.admin.ch/en/create/new?cube=https://agriculture.ld.admin.ch/foag/cube/MilkDairyProducts/Consumption_Price_Month&dataSource=Int).
2. Add another chart (e.g. line).
3. Proceed to layout options.
4. Switch to a dashboard layout.
5. ❌ See that it's not possible to enable shared Date filter.
